src/TurbulenceConvection/Grid.jl

```diff
@@ -0,0 +1,243 @@
+
```

Comment:
We shouldn't need this TurbulenceConvection. Mesh will be inherited from ClimaAtmos. If we want smaller domains in a SCM configuration of ClimaAtmos running the EDMF scheme, this should be configured in a ClimaAtmos driver.